### ISSUE TITLE
bug: flatpak override isn't working

### ISSUE THREAD

**Issue by 0xMRTT** [GradienceTeam/Gradience#574]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

In the welcome setup and in pref, the flatpak override isn't working at all.
I would also like to have polkit for applying as superuser

### Expected behavior

That there will be an automated override

### OS

Arch

### Installation method

Manual using meson

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tfuxu** [comment]
@0xMRTT 
In which circumstances those overrides functions doesn't work? On my machines I can't reproduce this, because everything is working as intended (even when creating a new unmodified virtual machines).

**Comment by 0xMRTT** [comment]
Hmm, for a lot of users (me include) it's not working

**Comment by 0xMRTT** [comment]
#575 maybe

**Comment by LyesSaadi** [comment]
Nope, not related I think... Here it's manual from meson, not flatpak package, according to you at least.

**Comment by LyesSaadi** [comment]
But, yeah, it's probable that some flathub users were affected, thus originating in some of the bugs reported about the override.

**Comment by 0xMRTT** [comment]
Oh yes, I'm using meson install so the Flatpak parameter will change nothing for me 😅.

**Comment by daudix-UFO** [comment]
I think it works now
